src/main/scala/org/ergoplatform/modifiers/history/BlockTransactions.scala

```diff
@@ -20,11 +21,13 @@ import scorex.util.Extensions._
 /**
   * Section of a block which contains transactions.
   *
-  * @param headerId - identifier of a header of a corresponding block
-  * @param txs - transactions of a block
-  * @param sizeOpt - (optional) size of the section (cached to not be calculated again)
+  * @param headerId     - identifier of a header of a corresponding block
+  * @param blockVersion - block version
```

Comment:
what is block version?

Reply:
protocol version for the bloc, changed desc a bit